assets/js/pages/AdvisoryDetails/AdvisoryDetails.test.jsx

```diff
@@ -27,39 +29,83 @@ describe('AdvisoryDetails', () => {
     render(<AdvisoryDetails advisoryName={advisoryName} errata={errata} />);
 
     expect(screen.getByText(advisoryName)).toBeVisible();
-    expect(screen.getByText(errata.synopsis)).toBeVisible();
-    expect(screen.getByText(errata.advisory_status)).toBeVisible();
-    expect(screen.getByText(errata.description)).toBeVisible();
+    expect(screen.getByText(errata.errata_details.synopsis)).toBeVisible();
+    expect(
+      screen.getByText(errata.errata_details.advisory_status)
+    ).toBeVisible();
+    expect(screen.getByText(errata.errata_details.description)).toBeVisible();
   });
 
-  it('displays CVEs, packages and fixes', () => {
+  it('packages', () => {
     const errata = advisoryErrataFactory.build();
     const advisoryName = faker.lorem.word();
 
-    const fixes = faker.word.words(2).split(' ');
     const packages = faker.word.words(2).split(' ');
-    const cves = faker.word.words(2).split(' ');
 
     render(
       <AdvisoryDetails
         advisoryName={advisoryName}
         errata={errata}
-        fixes={fixes}
-        cves={cves}
         packages={packages}
       />
     );
 
-    fixes.forEach((expectedWord) => {
+    packages.forEach((expectedWord) => {
       expect(screen.getByText(expectedWord)).toBeVisible();
     });
+  });
 
-    cves.forEach((expectedWord) => {
-      expect(screen.getByText(expectedWord)).toBeVisible();
+  it('displays fixes with according link', () => {
```

Comment:
Not sure what this means, what about something like

```suggestion
  it('displays fixes with valid link', () => {
```

Reply:
Yeah that fits better, thanks &&